libcxx/test/std/atomics/atomics.types.operations/atomics.types.operations.req/atomic_compare_exchange_strong.pass.cpp

```diff
@@ -6,7 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-// XFAIL: !non-lockfree-atomics
+// XFAIL: !has-64-bit-atomics
```

Comment:
I think those need to be `!has-128-bit-atomics` since they instantiate `atomic<LargeUserAtomicType>`, which contains `int [128]` -- that's larger than 64 bits. It's also larger than 128 bits but the check should work for both, I think.